nesta/production/orms/crunchbase_es_config.json

```diff
@@ -141,10 +141,22 @@
                     "type": "text"
                 },
                 "terms_roles_organisation": {
-                    "type": "keyword"
```

Comment:
As we discussed, this should become an array of `keyword`s #150

Reply:
This would be a separate PR, since this occurs in a different pipeline